tools/toolchain/scripts/arch_base.tmpl

```diff
@@ -10,6 +10,7 @@ WFLAGS      = ${WFLAGS}
 #
 FCDEBFLAGS  = ${FCDEBFLAGS}
 CFLAGS      = ${CFLAGS}
+CXXFLAGS    = ${CXXFLAGS}
```

Comment:
Does DeePMD-kit also offer C API or would it be easy to add? So, far we've managed to keep C++ out of CP2K and I'd really like to keep it that way.